src/content/smart-contracts/index.md

```diff
@@ -7,19 +7,13 @@ sidebar: true
 
 # Introduction to smart contracts {#introduction-to-smart-contracts}
 
-Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allows us to convert traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
+A smart contract is a digital contract that automatically executes the agreed on terms when its conditions are met. Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). Fundamentally, they are computer programs following "if X then Y" logic, which means they behave exactly as programmed and are resistant to participants' reneging on the terms of the contract. 
 
-Nick Szabo coined the term "smart contract". In 1994, he wrote [an introduction to the concept](https://www.fon.hum.uva.nl/rob/Courses/InformationInSpeech/CDROM/Literature/LOTwinterschool2006/szabo.best.vwh.net/smart.contracts.html) and, in 1996, [an exploration of what smart contracts could do](https://www.fon.hum.uva.nl/rob/Courses/InformationInSpeech/CDROM/Literature/LOTwinterschool2006/szabo.best.vwh.net/smart_contracts_2.html).
+In 1994, Nick Szazbo wrote [an introduction to the concept](https://www.fon.hum.uva.nl/rob/Courses/InformationInSpeech/CDROM/Literature/LOTwinterschool2006/szabo.best.vwh.net/smart.contracts.html) and, in 1996, [an exploration of what smart contracts could do](https://www.fon.hum.uva.nl/rob/Courses/InformationInSpeech/CDROM/Literature/LOTwinterschool2006/szabo.best.vwh.net/smart_contracts_2.html).
 
-Nick Szabo envisioned a digital marketplace built on these automatic, cryptographically secure processes. A place where transactions and business functions can happen trustlessly — without intermediaries. Smart contracts on Ethereum put this vision into practice.
+Szabo envisioned a digital marketplace where automatic, cryptographically-secure processes enable transactions and business functions to happen without trusted intermediaries. Smart contracts on Ethereum put this vision into practice.
 
-## What are contracts? {#what-are-contracts}
-
-You're probably thinking: _"I'm not a lawyer! Why would I care about contracts?"_. For most people, contracts bring to mind needlessly long terms and conditions agreements or boring legal documents.
-
-Contracts are just agreements. That is, any form of agreement can be encapsulated within the conditions of a contract. Verbal agreements or pen-and-paper contracts are acceptable for many things, but they aren't without flaws.
```

Comment:
A) I think you're assuming a bit much about what the audience is 'probably thinking' -- I personally think that most readers will be familiar with contracts and don't think of them as 'only for lawyers.' I recently signed a lease on an apartment, for example, without a lawyer. 

But also, aren't these developer docs? Developer docs don't need editorializing about the importance of the subject, IMO, which is effectively an attempt to sell them on why they should care about the subject. They've already come to the page to learn.

As to the second bit, I thought it was repetitive with other sections/unnecessary to the core idea. But I'm not dead set on that removal or anything 😃

Reply:
> A) I think we're assuming more than we need to about what the audience is 'probably thinking' -- I personally think that most readers will be familiar with contracts and don't think of them as 'only for lawyers.' I recently signed a lease on an apartment, for example. Contracts feel like just everyday life to me? IDK

This is fair. IIRC this was added after some minor user testing that indicated that users with little understanding of what a smart contract was even found the name ("smart contract") to be a little intimidating/corporate.

> But also, aren't these developer docs? Developer docs don't need to persuade the audience about why they should care about the subject, IMO, because they've already come to the page to learn.

No, these are what we call "Learn pages". Learn pages should be low on assumptions of knowledge, accessible to users without a strong technical understanding, and—as much as possible—without a need for an advanced understanding of the English language.

> As to the second bit, I thought it was repetitive with other sections/unnecessary to the core idea. But it's fine to include it 😃

Again, I'm in favour but I'd like some other opinions on this because I'm aware that I have some bias here.

Reply:
Thanks for the clarification, happy to defer on this and all matters

Reply:
Again, I think a compromise between these two options is best- delete the first paragraph, keep the second, IMO.

Reply:
going to remove these sentences as per the original PR because I think the point is now made well in later sections.